src/utils/azure.ts

```diff
@@ -88,6 +89,10 @@ export async function promptForStorageAccount(ui: IAzureUserInput, tree: AzureTr
         if (!result.keys || result.keys.length === 0) {
             throw new ArgumentError(result);
         } else {
+            if (saveStorageAccount && fsPath) {
```

Comment:
This logic feels unique to `runFromZipDeploy` and shouldn't be in promptForStorageAccount IMO

Reply:
While I agree that the logic is unique to runFromZipDeploy, there isn't an easy way to get access to the Storage Account's resource group and id.  Would you be opposed to me adding rg and id properties to the return value?

Reply:
That sounds like the perfect solution :)